thunderbird-development/codebase-overview/chat/message-styles.md

```diff
@@ -24,7 +24,7 @@ The other files are optional:
 
 | File Name | Fallback if missing | Usage |
 | :--- | :--- | :--- |
-| Footer.html | None \(will use an empty string\) | Displayed at the bottom of a conversation. This is the right place to \[\[Instantbird:Message\_Styles:JavaScript\|include JavaScript code\]\]. |
+| Footer.html | None \(will use an empty string\) | Displayed at the bottom of a conversation. |
```

Comment:
Should we now include 'inline.js' in this table?

Reply:
I decided not to because it says

> HTML files: these files are used to build the HTML markup of the conversation.

above the table. I guess it should be in the optional files list above though?

Reply:
I cannot answer those questions. You have to decide that :-)

Reply:
Maybe, there's a whole section on it below ("Including JavaScript in message styles") so maybe it is fine to just leave it as is.

Reply:
I'd argue if we want to add it that could still be done in a separate PR. As such this should be mergeable?

Reply:
That sounds fine with me. Let's merge it.